pkg/build/builder/dockerutil.go

```diff
@@ -113,3 +118,60 @@ func buildImage(client DockerClient, dir string, dockerfilePath string, noCache
 	}
 	return client.BuildImage(opts)
 }
+
+// dockerRun mimics the 'docker run' command. It uses the Docker Remote API to
+// create and start a container and stream its logs to stdout and stderr. The
+// container is removed after it terminates.
+func dockerRun(client DockerClient, createOpts docker.CreateContainerOptions, logsOpts docker.LogsOptions) error {
+	// Create a new container.
+	glog.V(4).Infof("Creating container ...")
```

Comment:
if this is not just your debugging, you should at least dump the createOpts to help debugging issues with container creation